### ISSUE TITLE
Test installation und use under Windows

### ISSUE THREAD

**Issue by johann-petrak** [GateNLP/python-gatenlp#171]
There are many different ways for how to install Python and python packages under Windows and then use the packages for development. 
Also a range of options for installing Java under Windows. Try out some of them and come up with a recommended way to do this.

**Comment by johann-petrak** [comment]
First approach:
* install miniconda: download from https://docs.conda.io/en/latest/miniconda.html Windows 64 bit and install: 
    * install for Just Me (recommended), default location
    * do NOT add miniconda to the PATH (default)
    * Register Miniconda as default Python (default)
    * this makes available: Anaconda Powershell Prompt and Anaconda Prompt
* create conda environment:
    * `conda create -y -n gatenlp python=3.9`
    * `conda activate gatenlp`
    * `python -m pip install gatenlp[all,notebook]`
* install GATE: https://gate.ac.uk/download/#latest 
    * use platform-specific installer for Windows
    * run it: prompts for downloading JDK
    * click OK - shows the adoptopenjdk.net page
    * shows OpenJDK 11 / Hotspot preselected
    * NOTE: shows that the page has moved to adoptium.net
        * on that page version 11 is preselected
        * download JDK .msi for x64
    * install Eclipse Temorin JDK Hotspot, 11.0.15 with default options
    * now restart GATE installation: prompts for java again: ?????
    * cannot find any new entry in the start menu, but when I re-run the Java installer it shows that it is already installed

**Comment by ianroberts** [comment]
> * now restart GATE installation: prompts for java again: ?????

The `exe` installer (as well as the `gate.exe` launcher) chooses a JVM by looking in the registry, in order for a non-Oracle Java to work you need to select a non-default option in the AdoptOpenJDK installer to set registry keys compatible with Oracle Java.  This is spelled out on the GATE download page

> The [AdoptOpenJDK](https://adoptopenjdk.net/) website provides packages for various operating systems, and is particularly suitable for Windows users - ensure you enable the "JavaSoft (Oracle) registry keys" option in the installer, otherwise gate.exe will not work

but I guess it's not clear here that the same applies to the installer `exe` as to `gate.exe`.

**Comment by ianroberts** [comment]
It may be that this option only exists in AdoptOpenJDK 8, not later versions.

**Comment by ianroberts** [comment]
Alternative might be [Zulu JDK](https://www.azul.com/downloads/) (the one that GATE recommends for Apple Silicon Macs) which definitely does install the Oracle-compatibility registry keys.

**Comment by johann-petrak** [comment]
Thanks Ian - I think I stumbled over this already once in the past - I am notoriously bad at reading the fine print and just expect things to just work with the default options :)
Just out of curiosity - why can't the GATE installer not also check for the location of the java command on the PATH (which apparently it is as i can run "java" from inside all the command terminals I have), at least as a fall-back option?

**Comment by ianroberts** [comment]
Does the adoptium/temurin installer for 11 still have the option to add JavaSoft registry keys?  If not maybe we should switch the recommendation on the GATE download page to Zulu across the board, which definitely does have that (though still as a non-default option).

**Comment by johann-petrak** [comment]
Yes, I just failed to initially activate it (the default is not to activate it).
However, one can run the installation program after the initial installation, choose to modify an existing installation and just select that option. After this, the GATE installer works fine. This is what I ended up doing.
